services/static-webserver/client/source/class/osparc/snapshots/IterationsView.js

```diff
@@ -119,14 +119,12 @@ qx.Class.define("osparc.snapshots.IterationsView", {
 
     __listenToNodeUpdates: function() {
       const socket = osparc.wrapper.WebSocket.getInstance();
-      const slotName = "nodeUpdated";
-      socket.on(slotName, data => {
-        const dataUpdate = JSON.parse(data);
-        const idx = this.__iterations.findIndex(it => it["uuid"] === dataUpdate["project_id"]);
+      socket.on("nodeUpdated", data => {
+        const idx = this.__iterations.findIndex(it => it.uuid === data["project_id"]);
```

Comment:
shouldn't we be consistent and use **data.project_id** instead of **data["project_id"]**?

Reply:
to my knowledge... dot access notation is only grammatically correct when keys are camel cased (JS standard)... maybe I am wrong